spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -748,7 +748,7 @@ public void open() {
       cursor = completionText.length();
     }
     if (Utils.isScala2_10()) {
-      ScalaCompleter c = (ScalaCompleter) Utils.invokeMethod(completor, "completor");
+      ScalaCompleter c = (ScalaCompleter) Utils.invokeMethod(completor, "completer");
```

Comment:
@karup1990 Thanks for fixing this.
Why don't you change another `completor` in this line? I mean
```
ScalaCompleter c = (ScalaCompleter) Utils.invokeMethod(completer, "completer");
```

Then you have to change [spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java#L114](https://github.com/apache/zeppelin/blob/master/spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java#L114) & [spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java#L116](https://github.com/apache/zeppelin/blob/master/spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java#L116) too.